internal/models/cluster/tkgaws/tkgaws_spec.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package tkgawsmodel
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// VmwareTanzuManageV1alpha1ClusterInfrastructureTkgawsSpec TKG AWS cluster spec.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.infrastructure.tkgaws.Spec
+type VmwareTanzuManageV1alpha1ClusterInfrastructureTkgawsSpec struct {
```

Comment:
`VmwareTanzuManageV1alpha1ClusterInfrastructureTkgawsSpec ` -> `VmwareTanzuManageV1alpha1ClusterInfrastructureTKGAWSSpec`

Reply:
The spec is auto-generated using the go swagger tool, these variables are not manually named here.